packages/rum-core/src/domain/rumEventsCollection/view/viewCollection.ts

```diff
@@ -26,6 +26,7 @@ function processViewUpdate(view: ViewEvent) {
     date: preferredTimeStamp(view.startClocks),
     type: RumEventType.VIEW,
     view: {
+      id: view.id,
```

Comment:
Adding the id here does not seem useful since it will be overridden later
what about not adding it?

Reply:
I can't see where it is overriden?